### PR TITLE
minor improvements

### PR DESCRIPTION
**Pull request by andrzejnovak**
- autoMCstat to have `lnN` option for quick fits
- protect autoMCstats when only one low bin is filled
  - down variation results in 0 norm
- render combine model with same names
  - each model has it's own dir anyway, can reuse commands more easily

### CODE REVIEW

**Comment by andrzejnovak** [comment]
@nsmith-

File: rhalphalib/model.py [hunk 1]
```diff
@@ -192,18 +192,19 @@ def getObservation(self):
         Return the current observation set for this Channel as plain numpy array
         If it is non-poisson, it will be returned as a tuple (sumw, sumw2)
         '''
+        _mask_val = float(0)
```
Comment (nsmith-) [inline rhalphalib/model.py:195]: Would this make more sense as a class member? Also what's the use case for non-zero?

File: rhalphalib/sample.py [hunk 2]
```diff
@@ -474,9 +490,11 @@ def getExpectation(self, nominal=False):
         Create an array of per-bin expectations, accounting for all nuisance parameter effects
             nominal: if True, calculate the nominal expectation (i.e. just plain numbers)
         '''
+        # _mask_val = float(1e-20)
```
Comment (nsmith-) [inline rhalphalib/sample.py:493]: Stray comment

**Review by nsmith- — APPROVED** [review]

File: rhalphalib/model.py (hunk 1 above)
Comment (andrzejnovak) [reply rhalphalib/model.py]: Done. Use case was for debugging, where combine seemed to complain about 0 bins vs small epsilon.

**Review by nsmith- — APPROVED** [review]

**Pull request merged**